app/src/main/java/com/cmput301f20t21/bookfriends/ui/login/CreateAccountActivity.java

```diff
@@ -0,0 +1,175 @@
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_create);
+        this.getSupportActionBar().hide();
+        // hides the status bar, deprecated in API 30
+        View decorView = this.getWindow().getDecorView();
+        int uiOptions = View.SYSTEM_UI_FLAG_FULLSCREEN;
+        decorView.setSystemUiVisibility(uiOptions);
+
+        usernameLayout = findViewById(R.id.login_username_layout);
+        usernameField = (TextInputEditText) usernameLayout.getEditText();
+        usernameField.addTextChangedListener(new AfterTextChangedWatcher() {
+            @Override
+            public void afterTextChanged(Editable usernameString) {
+                // remove the error if any exist
+                usernameLayout.setError(null);
+            }
+        });
+
+        passwordLayout = findViewById(R.id.login_password_layout);
+        passwordField = (TextInputEditText) passwordLayout.getEditText();
+        passwordField.addTextChangedListener(new AfterTextChangedWatcher() {
+            @Override
+            public void afterTextChanged(Editable passwordString) {
+                // remove the error if any exist
+                passwordLayout.setError(null);
+            }
+        });
+
+        confirmLayout  = findViewById(R.id.Comfirm_Password_Layout);
+        confirmField = (TextInputEditText)confirmLayout.getEditText();
+        confirmField.addTextChangedListener(new AfterTextChangedWatcher() {
+            @Override
+            public void afterTextChanged(Editable comfirmString) {
+                // remove the error if any exist
+                confirmLayout.setError(null);
+            }
+        });
+
+        emailLayout = findViewById(R.id.email_layout);
+        emailField = (TextInputEditText)emailLayout.getEditText();
+        emailField.addTextChangedListener(new AfterTextChangedWatcher() {
+            @Override
+            public void afterTextChanged(Editable emailString) {
+                // remove the error if any exist
+                emailLayout.setError(null);
+            }
+        });
+
+        phoneLayout  = findViewById(R.id.phone_layout);
+        phoneField = (TextInputEditText) phoneLayout.getEditText();
+        phoneField.addTextChangedListener(new AfterTextChangedWatcher() {
+            @Override
+            public void afterTextChanged(Editable phoneNumber) {
+                // remove the error if any exist
+                phoneLayout.setError(null);
+            }
+        });
+
+    }
+    /**
+     * check the username field for emptiness, adding error message if field is empty
+     * @param username - the username that the user entered
+     * @return true if the username is not empty, false if username is empty
+     */
+    private boolean checkUsername(String username) {
+        boolean isUsernameEmpty = username.isEmpty();
+        if (isUsernameEmpty) {
+            usernameLayout.setError("Username can not be empty");
```

Comment:
Use string resource instead of hardcoded string

same for all the other fields